docs/guides/publish_fetch_packages.md

```diff
@@ -0,0 +1,177 @@
+Similarly as the {{open_aea}} framework, the {{open_autonomy}} framework also works with the concept of **package**: a collection of files that implement a specific component or functionality.
```

Comment:
Why are you referring to open_aea here? Most users should not need to know it even exists. At a minimum, I'd not start off referring to it as it creates unnecessary mental overhead.

Reply:
fixed